datafusion/common/src/error.rs

```diff
@@ -47,56 +47,82 @@ pub type GenericError = Box<dyn Error + Send + Sync>;
 #[derive(Debug)]
 pub enum DataFusionError {
     /// Error returned by arrow.
+    ///
     /// 2nd argument is for optional backtrace
     ArrowError(ArrowError, Option<String>),
-    /// Wraps an error from the Parquet crate
+    /// Error when reading Parquet data.
```

Comment:
Is this only for when reading parquet, or also when writing?

Reply:
I think you are correct -- it also applies to writing.

```suggestion
    /// Error when reading / writing Parquet data.
```